python/ccf/clients.py

```diff
@@ -369,26 +367,29 @@ def request(self, request, is_signed=False):
 
 
 class WSClient:
-    def __init__(
-        self,
-        host,
-        port,
-        cert=None,
-        key=None,
-        ca=None,
-        request_timeout=DEFAULT_REQUEST_TIMEOUT_SEC,
-        **kwargs,
-    ):
+    """
+    CCF WebSocket client implementation.
+
+    Warning: Does not handle client signatures.
```

Comment:
Client signatures over WS aren't supported in CCF, so I think this should either be clarified or removed, but as it stands it sounds like it's a limitation of the client.

Reply:
Good point. I've re-phrased the now-note.